lib/server/controllers/logs.controller.js

```diff
@@ -320,6 +320,44 @@ const getRootLogsHandler = async (request, response) => {
     }
 };
 
+/**
+ * Get all child logs by parent id
+ *
+ * @param {Object} request The *request* object represents the HTTP request and has properties for the request query
+ *                         string, parameters, body, HTTP headers, and so on.
+ * @param {Object} response The *response* object represents the HTTP response that an Express app sends when it gets an
+ *                          HTTP request.
+ * @returns {undefined}
```

Comment:
We don't define all this in controller anymore, we simply put what the route does (what you did well line 324) and we mark it as ignored by eslint. Please have a look in `dplProcess.controller.js`

Reply:
Done